test/testdata/app-scenarios/docker-compose/complex/docker-compose.imported.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 items:
-- apiVersion: v1
+- apiVersion: image.openshift.io/v1
```

Comment:
@smarterclayton this might be specific to compose code, you don't have to do this for any other command.